docker/docker-compose.yml

```diff
@@ -58,6 +59,11 @@ services:
     container_name: converters_container
     ports:
       - "${CONVERTER_PORT:-3082}:8080"
+  semantic:
+    image: pathwaycommons/semantic-search:${SEMANTIC_SEARCH_IMAGE_TAG:-latest}
+    container_name: semantic_container
+    ports:
```

Comment:
Remove `ports`. It will start to get very confusing on the server.

Reply:
I did that (https://github.com/PathwayCommons/factoid/pull/727/commits/c62a5f23c152beca527b1a2842528775f492f29d). Interestingly, the problem that I faced in docker-compose is also gone when stop port forwarding by using ``export`` instead of ``ports``.